frontend/src/routes/(authenticated)/project/[project_code]/OrgList.svelte

```diff
@@ -23,9 +32,35 @@
       </div>
     {/if}
     {#each organizations as org (org.id)}
-      <Badge>
-        {org.name}
-      </Badge>
+      {#if !canManage}
+        <Badge>
+          {org.name}
+        </Badge>
+      {:else}
+        <Dropdown>
+          <ActionBadge actionIcon="i-mdi-dots-vertical" on:action>
+            <span class="pr-3 whitespace-nowrap overflow-ellipsis overflow-x-clip" title={org.name}>
+              {org.name}
+            </span>
+          </ActionBadge>
+          <ul slot="content" class="menu">
+            <li>
+              <mj-button href={`/org/${org.id}`}>
```

Comment:
we shouldn't be using an mj-button here, that's only valid inside an email template. instead it should probably be an `a` tag